commander/src/bootstrapping/commands/keys/create.ts

```diff
@@ -0,0 +1,164 @@
+			default: false,
+		}),
+		password: flagsWithParser.password,
+		count: flagParser.integer({
+			char: 'c',
+			description: 'Number of keys to create',
+			default: 1,
+		}),
+		offset: flagParser.integer({
+			char: 'f',
+			description: 'Offset for the key derivation path',
+			default: 0,
+		}),
+		chainid: flagParser.integer({
+			char: 'i',
+			description: 'Chain id',
+			default: 0,
+		}),
+	};
+
+	async run(): Promise<void> {
+		const {
+			flags: {
+				output,
+				passphrase: passphraseSource,
+				'no-encrypt': noEncrypt,
+				password: passwordSource,
+				count,
+				offset,
+				chainid,
+			},
+		} = this.parse(CreateCommand);
+
+		if (output) {
+			const { dir } = path.parse(output);
+			fs.ensureDirSync(dir);
+		}
+		const passphrase = passphraseSource ?? (await getPassphraseFromPrompt('passphrase', true));
+
+		const keys = [];
+		for (let i = 0; i < count; i += 1) {
+			const accountKeyPath = `m/44'/134'/${offset + i}'`;
+			const generatorKeyPath = `m/25519'/134'/${chainid}'/${offset + i}'`;
+			const blsKeyPath = `m/12381/134/${chainid}/${offset + i}`;
+
+			const accountPrivateKey = await ed.getKeyPairFromPhraseAndPath(passphrase, accountKeyPath);
+			const accountPublicKey = ed.getPublicKeyFromPrivateKey(accountPrivateKey);
+			const address = addressUtil.getAddressFromPublicKey(accountPublicKey);
+			const generatorPrivateKey = await ed.getKeyPairFromPhraseAndPath(
+				passphrase,
+				generatorKeyPath,
+			);
+			const generatorPublicKey = ed.getPublicKeyFromPrivateKey(generatorPrivateKey);
+			const blsPrivateKey = await bls.getPrivateKeyFromPhraseAndPath(passphrase, blsKeyPath);
+			const blsPublicKey = bls.getPublicKeyFromPrivateKey(blsPrivateKey);
+
+			let password: string;
+			let encryptedMessageObject = {};
+			if (!noEncrypt) {
+				password = passwordSource ?? (await getPasswordFromPrompt('password', true));
+				const plainGeneratorKeyData = {
+					generatorKey: generatorPublicKey,
+					generatorPrivateKey,
+					blsKey: blsPublicKey,
+					blsPrivateKey,
+				};
+				const encodedGeneratorKeys = codec.encode(plainGeneratorKeysSchema, plainGeneratorKeyData);
+				encryptedMessageObject = await encrypt.encryptMessageWithPassword(
+					encodedGeneratorKeys,
+					password,
+				);
+			}
+
+			keys.push({
+				address: address.toString('hex'),
+				keyPath: accountKeyPath,
+				publicKey: accountPublicKey.toString('hex'),
+				privateKey: accountPrivateKey.toString('hex'),
+				plain: {
+					generatorKeyPath,
+					generatorKey: generatorPublicKey.toString('hex'),
+					generatorPrivateKey: generatorPrivateKey.toString('hex'),
+					blsKeyPath,
+					blsKey: blsPublicKey.toString('hex'),
+					blsProofOfPosession: bls.popProve(blsPrivateKey).toString('hex'),
+					blsPrivateKey: blsPrivateKey.toString('hex'),
+				},
+				encrypted: encryptedMessageObject,
+			});
+		}
+
+		if (output) {
+			fs.writeJSONSync(output, JSON.stringify({ keys }), { spaces: ' ' });
```

Comment:
this is writing the file as `string`. I think you meat to put the `keys` object without `stringify`

Reply:
Quick question, if it's written to file as object how to parse when it's read back, because I think `fs.readFileSync` returns only string ?